packages/gestalt-design-tokens/tokens/vr-theme/sema/rounding.json

```diff
@@ -1,54 +1,49 @@
 {
   "sema": {
     "rounding": {
-      "date": {
-        "container": {
-          "type": "other",
-          "value": "{base.rounding.16}"
-        },
-        "range": {
-          "middle": {
-            "type": "other",
-            "value": "{base.rounding.0}"
-          }
-        },
-        "start": {
-          "type": "other",
-          "value": "{base.rounding.circle} {base.rounding.0} {base.rounding.0} {base.rounding.circle}"
-        },
-        "end": {
-          "type": "other",
-          "value": "{base.rounding.0} {base.rounding.circle} {base.rounding.circle} {base.rounding.0}"
-        }
-      },
-      "XL": {
-        "type": "other",
-        "value": "{base.rounding.20}"
-      },
-      "LG": {
-        "type": "other",
-        "value": "{base.rounding.16}"
-      },
-      "MD": {
-        "type": "other",
-        "value": "{base.rounding.12}"
-      },
-      "SM": {
-        "type": "other",
-        "value": "{base.rounding.8}"
-      },
-      "XS": {
-        "type": "other",
-        "value": "{base.rounding.4}"
-      },
-      "circleappenda": {
-        "$extensions": {
-          "studio.tokens": {
-            "modify": {}
-          }
-        },
-        "type": "other",
-        "value": "{base.rounding.circle}"
+      "0": {
```

Comment:
did we document that we have semnantic only tokens with no base when there's no both base and semantic? is this to prevent duplication and expose only semantic?

Reply:
Yes -  we reviewed this change in the 2024 doc. Ultimately, we want to use the `unit` to derive the sema tokens, and avoid dupes